scout/server/blueprints/variant/utils.py

```diff
@@ -181,7 +181,7 @@ def add_gene_info(store, variant_obj, gene_panels=None, genome_build=None):
         panel_info = add_panel_specific_gene_info(extra_info.get(hgnc_id, []))
         variant_gene.update(panel_info)
 
-        update_transcripts_information(variant_gene, hgnc_gene, variant_obj)
+        update_transcripts_information(variant_gene, hgnc_gene, variant_obj, genome_build)
 
```

Comment:
Could you add a docstring here? (original suggestion by @dnil)

Reply:
Ehm I don't think it was here that he meant!

Reply:
Almost, it was for this function at least.. 😸 Look where it is defined!

Reply:
Yeah I fixed it I think!